src/RestServer/RestServerPlugin.Console.cs

```diff
@@ -0,0 +1,131 @@
+                ConsoleHelper.Info("   Name: ", wallet.Name ?? "\"\"");
+                ConsoleHelper.Info("   Path: ", wallet.Path);
+                ConsoleHelper.Info("Version: ", wallet.Version.ToString());
+                ConsoleHelper.Info("Session: ", key.ToString("n"));
+                ConsoleHelper.Info("Expires: ", Math.Round(expires.TotalSeconds, 0).ToString(), " second(s).");
+
+                if (WalletController.WalletSessions.Count > 1)
+                    ConsoleHelper.Info();
+            }
+
+            if (WalletController.WalletSessions.Any())
+                ConsoleHelper.Info("--------------------------");
+
+            ConsoleHelper.Info("  Total: ", WalletController.WalletSessions.Count.ToString(), " session(s).");
+        }
+
+        [ConsoleCommand("rest kill", Category = "RestServer Commands", Description = "Kills an active wallet session.")]
+        private void OnKillWalletSession(string sessionId)
+        {
+            if (Guid.TryParse(sessionId, out var session) == false)
+            {
+                ConsoleHelper.Warning("Invalid session id.");
+                return;
+            }
+
+            ConsoleHelper.Info($"You are about to kill ", session.ToString("n"), " session!");
+            var answer = ReadUserInput($"Are you sure?");
+            if (answer.Equals("yes", StringComparison.InvariantCultureIgnoreCase) || answer.Equals("y", StringComparison.InvariantCultureIgnoreCase))
+            {
+                if (WalletController.WalletSessions.TryRemove(session, out _))
+                    ConsoleHelper.Info("Session ", session.ToString("n"), " has terminated.");
+                else
+                    ConsoleHelper.Error($"Session {session:n} could not be terminated. Try again later.");
+            }
+        }
+
+        [ConsoleCommand("rest killall", Category = "RestServer Commands", Description = "Kills all active wallet sessions.")]
+        public void OnKillallWalletSessions()
+        {
+            if (WalletController.WalletSessions.Any() == false)
+            {
+                ConsoleHelper.Info("No ", "active", " sessions.");
+                return;
+            }
+
+            var answer = ReadUserInput($"Kill all active wallet sessions?");
+            if (answer.Equals("yes", StringComparison.InvariantCultureIgnoreCase) || answer.Equals("y", StringComparison.InvariantCultureIgnoreCase))
+            {
+                foreach (var (session, _) in WalletController.WalletSessions)
+                {
+                    if (WalletController.WalletSessions.TryRemove(session, out _))
+                        ConsoleHelper.Info("Session ", session.ToString("n"), " has terminated.");
+                    else
+                        ConsoleHelper.Error($"Session {session:n} could not be terminated. Try again later.");
+                }
+            }
+        }
+
+        private string ReadUserInput(string prompt, bool password = false)
```

Comment:
Move this method to `ConsoleHelper`?

Reply:
It would be better if `Neo.ConsoleService` made it `public`. It's the same one i copied it from there.